style.css

```diff
@@ -283,3 +298,280 @@ footer {
   margin-left: auto;
   margin-right: auto;
 }
+
+.sectionHeader {
+  display: flex;
+  align-items: flex-end;
+}
+
+.headerX {
+  display: flex;
+  justify-content: center;
+}
+
+.huddle {
+  display: flex;
+  margin-left: 12%;
+  margin-bottom: 12%;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.divButton {
+  margin-left: 331%;
+}
+
+@media screen and (max-width: 1440px) {
+  header {
+    height: 179vh;
```

Comment:
anche quail vh non ha senso. Non devi per forza dare altezze fisse agli elementi comunque, per alcuni elementi puoi anche far si che la grandezza sia in base al contenuto